docs/GettingStartedDocs/Contributors/BuildingInADockerContainer.md

```diff
@@ -0,0 +1,32 @@
+# Getting Started with building Open Enclave in a Docker Container
+
+## Platform requirements
+
+- Linux distribution capable of running Docker containers.
+
+## Steps to building Open Enclave inside of a Docker container
+
+1. Install Docker CE by following these instructions: https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce-1
```

Comment:
I think `https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce` is more appropriate which include instructions on how to enable docker repository.

Reply:
Will make this change, thanks!